README.md

```diff
@@ -160,20 +160,20 @@ USAGE:
     snarkos start [OPTIONS]
 
 OPTIONS:
-        --beacon <BEACON>          Specify this as a beacon, with the given account private key for this node
-        --client <CLIENT>          Specify this as a client, with an optional account private key for this node
-        --connect <CONNECT>        Specify the IP address and port of a peer to connect to [default: ]
-        --dev <DEV>                Enables development mode, specify a unique ID for this node
-    -h, --help                     Print help information
-        --logfile <LOGFILE>        Specify the path to the file where logs will be stored [default: /tmp/snarkos.log]
-        --network <NETWORK>        Specify the network of this node [default: 3]
-        --node <NODE>              Specify the IP address and port for the node server [default: 0.0.0.0:4133]
-        --nodisplay                If the flag is set, the node will not render the display
-        --norest                   If the flag is set, the node will not initialize the REST server
-        --prover <PROVER>          Specify this as a prover, with the given account private key for this node
-        --rest <REST>              Specify the IP address and port for the REST server [default: 0.0.0.0:3033]
-        --validator <VALIDATOR>    Specify this as a validator, with the given account private key for this node
-        --verbosity <VERBOSITY>    Specify the verbosity of the node [options: 0, 1, 2, 3] [default: 2]
+        --beacon <PRIVATE KEY>           Specify this as a beacon, with the given account private key for this node as an argument
+        --client <PRIVATE KEY>           Specify this as a client, with an optional account private key for this node as an argument
```

Comment:
This clarifies what should be here, but people running `snarkos start --help` tool would still display the original verbage. It might be better if changed the node type to a flag and had --private_key as an option.

```suggestion
        --beacon <PRIVATE KEY>           Specify this as a beacon, with the given account private key for this node as an argument
        --client <PRIVATE KEY>           Specify this as a client, with an optional account private key for this node as an argument
```